ui/component/or-map/src/mapwidget.ts

```diff
@@ -873,6 +911,15 @@ export class MapWidget {
         }
     }
 
+    public async getGeoJsonFeature(query: {query: string}){
+        // let gc: MaplibreGeocoder. = this._geocoder;
+    }
```

Comment:
Does this not work? Can it be fixed or removed or at least commented out with reason